WindowsMRAssetConverter/WindowsMRAssetConverter.cpp

```diff
@@ -226,21 +233,26 @@ int wmain(int argc, wchar_t *argv[])
                 auto lod = lodFilePaths[i];
                 auto subFolder = FileSystem::CreateSubFolder(tempDirectory, L"lod" + std::to_wstring(i + 1));
 
-                lodDocuments.push_back(LoadAndConvertDocumentForWindowsMR(lod, AssetTypeUtils::AssetTypeFromFilePath(lod), subFolder, maxTextureSize, packing, !shareMaterials, !replaceTextures, meshCompression));

```

Comment:
Are we processing the materials multiple times if the shareMaterials flag is on?

Reply:
Previously the top level would have the materials processed, then each loaded lod would perform the same steps as they were loaded _unless_ the shareMaterials flag was set (since in that case the material references would be updated to use the top level ones anyways).

The issue I'm trying to solve here is that the comparison that was being made with the shareMaterials flag set started to fail now that the top level materials were converted from sg -> mr.

I'm just putting the material processing step afterward the lod merging, instead of running it as part of the initial load.